RoutesApp_ios/Assets.xcassets/Icons/arrow-left-primary.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Arrow 1.svg",
```

Comment:
use more descriptive names for your files, Arrow 1.svg should be arrow-left-primary.svg